NSS-Day-Cohort-69/tabloid-one-more-div
# Bugfix - Rerender on Approve

### Purpose 
To have the Unapproved Posts page rerender after a Post was approved

### Files Changed
- Updated onClick logic for "Approve" buttons to properly rerender the page in `ApprovePost.jsx`
- Updated rendering to look correct when a Post has no Tags in `PostDetails.jsx`

### To Test
Fetch, setup, and run according to project README, then confirm the following
- Clicking "Approve" on an unapproved Post in the Unapproved Posts List view causes the page to rerender, with the now approved Post no longer appearing

closes #73
